### ISSUE TITLE
UserWarning: Gradients do not exist for variables ['sub_tkan_kernel'] when minimizing the loss

### ISSUE THREAD

**Issue by mw66** [remigenet/TKAT#8]
with KERAS_BACKEND=torch

/lib/python3.11/site-packages/keras/src/optimizers/base_optimizer.py:678: UserWarning: Gradients do not exist for variables ['sub_tkan_kernel'] when minimizing the loss. If using `model.compile()`, did you forget to provide a `loss` argument?

**Comment by remigenet** [comment]
It may be linked to https://github.com/remigenet/TKAT/issues/9#issuecomment-2413096820, could you look if the change solve the issue or otherwise give me setup and reproductible example ?

**Comment by remigenet** [comment]
Closing as stale
